### PR TITLE
Redisify store adjustment

### PR DESCRIPTION
**Pull request by 1x2kb**

### CODE REVIEW

File: routes/eth-lookup.js [hunk 1]
```diff
@@ -66,12 +65,20 @@ async function getUrl(name) {
...
+    };
   }
 
   const lookupObject = await doLookup(name);
-  if (lookupObject) 
-    await saveNameUrl(lookupObject);
+  if (lookupObject) {
+    let minutes = process.env.REDIS_EXPIRATION_MINUTES
+    if (typeof minutes === "string")
+      minutes = parseInt(minutes);
```
Comment (bradrf) [inline routes/eth-lookup.js:78]: should we have an `else minutes = 5` here?

**Review by bradrf — APPROVED** [review]
other than the default ttl case, looks good to me!

File: routes/eth-lookup.js (hunk 1 above)
Comment (1x2kb) [reply routes/eth-lookup.js:78]: It is default assigned to 5 so we shouldn't need it unless you thought of something I didnt.

Comment (bradrf) [reply routes/eth-lookup.js:78]: oh! nope. i missed that. looks good to m!

**Review by bradrf — APPROVED** [review]

**Pull request merged**